server/index.js

```diff
@@ -28,7 +29,55 @@ app.use('/', expressStaticGzip(__dirname + '/public', {
 // a route to request all review objects from the db
 app.get('/reviews/:id', (req, res) => {
   const id = req.params.id;
-  Review.find({dbId: id})
+  Review.find({'reviews.username': "AlexK"})
```

Comment:
"AlexK" is hardcoded.  Did you mean to leave that there, or was that for testing?

Reply:
Yes, that was for testing. I think I will reseed my DB many times in future.